shared/process/spawn.rb

```diff
@@ -469,7 +469,7 @@
 
   it "raises an Errno::EACCES when the file does not have execute permissions" do
     lambda { @object.spawn __FILE__ }.should raise_error(Errno::EACCES)
-  end
+  end unless File.executable?(__FILE__) # Some FS (e.g. vboxfs) locate all files executable
```

Comment:
Same here of course :wink: 
Probably something that should be part of the style guide (#50) though.